### ISSUE TITLE
Fatal Error with WP GraphQL 0.9.0

### ISSUE THREAD

**Issue by cjkoepke** [pristas-peter/wp-graphql-gutenberg#38]
After installing the following:

1. [wp-graphql@0.9.0](https://github.com/wp-graphql/wp-graphql/tree/v0.9.0)
2. [wp-graphql-gutenberg@0.2.1](https://github.com/pristas-peter/wp-graphql-gutenberg/tree/v0.2.1)

I get a 500 error when accessing the `/graphql` endpoint:

```
Uncaught ArgumentCountError: Too few arguments to function register_graphql_scalar(), 1 passed in /var/www/html/wp-content/plugins/wp-graphql-gutenberg-0.2.1/src/Schema/Types/Scalar/Scalar.php on line 31 and exactly 2 expected in /var/www/html/wp-content/plugins/wp-graphql-0.9.0/access-functions.php:254

Stack trace:

#0 /var/www/html/wp-content/plugins/wp-graphql-gutenberg-0.2.1/src/Schema/Types/Scalar/Scalar.php(31): register_graphql_scalar(Array)

#1 /var/www/html/wp-content/plugins/wp-graphql-gutenberg-0.2.1/src/Schema/Types/BlockTypes.php(69): WPGraphQLGutenberg\Schema\Types\Scalar\Scalar::BlockAttributesArray()

#2 /var/www/html/wp-content/plugins/wp-graphql-gutenberg-0.2.1/src/Schema/Types/BlockTypes.php(97): WPGraphQLGutenberg\Schema\Types\BlockTypes::get_attribute_type('description', Array, 'YoastHowToBlock...')

#3 /var/www/html/wp-content/plugins/wp-graphql-gutenberg-0.2.1/src/Schema/Types/BlockTypes.php(143): WPGraphQLGutenberg\Schema\Types\BlockTypes::create_attributes_fields(Array, 'YoastHowToBlock.. in /var/www/html/wp-content/plugins/wp-graphql-0.9.0/access-functions.php on line 254
```

**Comment by pristas-peter** [comment]
Fixed in v0.2.2

**Comment by pristas-peter** [comment]
Fixed in 370d5386bd1c19d0304dcedf3f6823ce08a93b25
